horreum-backend/src/test/java/io/hyperfoil/tools/horreum/svc/RunServiceTest.java

```diff
@@ -174,4 +235,26 @@ private String testDataSetQuery(String jsonPath, boolean array, String schemaUri
       }
       return value;
    }
+
+   private Transformer createExampleTransformerWithJsonPath(String name, Schema schema, NamedJsonPath... paths) {
+      Transformer transformer = new Transformer();
+      transformer.name = name;
+      transformer.extractors = new ArrayList<>();
+      for (NamedJsonPath path : paths) {
+         transformer.extractors.add(path);
+      }
+      transformer.owner = TESTER_ROLES[0];
+      transformer.access = Access.PUBLIC;
+      transformer.schema = schema;
```

Comment:
It is not necessary to set schema here, it should be ignored for purposes of JSON transformation

Reply:
My bad, actually the serializer would throw NPE without it